scripts/drone/steps/lib.star

```diff
@@ -1023,3 +1024,21 @@ def validate_scuemata_step():
             './bin/linux-amd64/grafana-cli cue validate-schema --grafana-root .',
         ],
     }
+
+def ensure_cuetsified_step():
+    return {
+        'name': 'ensure-cuetsified',
+        'image': build_image,
+        'depends_on': [
+            'validate-scuemata',
+        ],
+        'commands': [
+            './bin/linux-amd64/grafana-cli cue gen-ts --grafana-root .',
+            '# The above command generates Typescript files (*.gen.ts) from all appropriate .cue files.',
+            '# It is required that the generated Typescript be in sync with the input CUE files.',
+            '# ...Modulo eslint auto-fixes...:',
+            './node_modules/.bin/eslint . --ext .gen.ts --fix',
+            '# If any filenames are emitted by the below script, run the generator command locally and commit the result.',
```

Comment:
🤔 It can be repeating ourselves here, but I think we could add the actual command on the comment, then is easier to just copy and paste it in the terminal
```suggestion
            '# If any filenames are emitted by the below script, run the generator command `grafana-cli cue gen-ts` locally and commit the result.',
```

Reply:
sure!